### PR TITLE
add unit test for helper.go

### PR DESCRIPTION
**Pull request by nayihz**
**What type of PR is this?**

> /kind cleanup

**What this PR does / why we need it**:
add unit test for helper.go

**Which issue(s) this PR fixes**:
add unit test

**Does this PR introduce a user-facing change?**:
```
NONE
```
**release-note**
```release-note
NONE
```
**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:
NO

### CODE REVIEW

**Comment by oomichi** [comment]
@latteczy Did you sign CLA for contributing?
If not, please see https://git.k8s.io/community/CLA.md#the-contributor-license-agreement to sign the CLA.

/release-note-none
/kind cleanup
/triage accepted

**Comment by latteczy** [comment]
> @latteczy Did you sign CLA for contributing?
> If not, please see https://git.k8s.io/community/CLA.md#the-contributor-license-agreement to sign the CLA.
> 
> /release-note-none
> /kind cleanup
> /triage accepted

done

**Review by oomichi — COMMENTED** [review]
Thanks for doing this.

/lgtm

**Comment by latteczy** [comment]
hi, @oomichi could anybody add `release-note-label` to this pr? Thanks.

**Comment by oomichi** [comment]
> hi, @oomichi could anybody add release-note-label to this pr? Thanks.

This just adds unit test, I think we don't need to write any release-note for this.
However you didn't write
```
release-note
NONE
```
on this PR message and the CI bot added `do-not-merge/release-note-label-needed` label.
Then we needed to run a command `/release-note-none` to remove the label.

**Comment by latteczy** [comment]
I have modified the PR description according to [release-note guide](https://github.com/kubernetes/community/blob/master/contributors/guide/release-notes.md#applying-a-release-note) and other merged pr. Please let me know if there is something wrong or unclear. Thanks very much.
@oomichi

**Comment by latteczy** [comment]
@oomichi Sorry to disturb you. This PR has been opened for a long time. Now it lacks an `approved` label. Could CI robot or somebody add this lable for this PR? Or Do I need to do something? Thanks for your help.

File: pkg/apis/core/v1/helper/helpers_test.go [hunk 1]
```diff
@@ -1509,3 +1509,98 @@ func TestHugePageUnitSizeFromByteSize(t *testing.T) {
...
+		left      *v1.LoadBalancerStatus
+		right     *v1.LoadBalancerStatus
+		name      string
+		expectVal bool
+	}{
+		{
+			name: "left equals right",
+			left: &v1.LoadBalancerStatus{
+				Ingress: []v1.LoadBalancerIngress{
+					{
```
Comment (thockin) [inline pkg/apis/core/v1/helper/helpers_test.go:1525]: small nit: you can linewrap this into the previous and save one indent level :)

File: pkg/apis/core/v1/helper/helpers_test.go [hunk 2]
```diff
@@ -1509,3 +1509,98 @@ func TestHugePageUnitSizeFromByteSize(t *testing.T) {
...
+						IP:       "1.1.1.1",
+						Hostname: "host1",
+					}},
+			},
+			expectVal: false,
+		},
+	}
+
+	for _, tc := range testCases {
+		t.Run(fmt.Sprintf("left input=%v, right input=%v, expected value=%v", tc.left, tc.right, tc.expectVal), func(t *testing.T) {
```
Comment (thockin) [inline pkg/apis/core/v1/helper/helpers_test.go:1599]: Don't sprintf into the Run name.  I think it's better to do something like:

```
t.Run(tc.name, func(t *testing.T) {
  //...
  t.Errorf("got %v, expected %v", v, tc.expectVal)

File: pkg/apis/core/v1/helper/helpers_test.go (hunk 1 above)
Comment (latteczy) [reply pkg/apis/core/v1/helper/helpers_test.go:1525]: The test casees in this file were all wrote in this style. I can format it to save one indent level.

Comment (latteczy) [reply pkg/apis/core/v1/helper/helpers_test.go]: This is a big change. I think I can format it in a new PR if it's worth to do so.

**Review by thockin — COMMENTED** [review]
Thanks!

/lgtm
/approve

**Pull request merged**